charts/connectivity-monitor/templates/ds.yaml

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: connectivity-exporter-config
+  namespace: {{ .Release.Namespace }}
+  labels: {app: connectivity-exporter}
+data:
+  filtered-ports: "443,6443"
+  filtered-ips: "0.0.0.0/0"
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: connectivity-exporter
+  namespace: {{ .Release.Namespace }}
+  labels: {app: connectivity-exporter}
+spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 100%
+
+  selector: {matchLabels: {app: connectivity-exporter}}
+  template:
+    metadata: {labels: {app: connectivity-exporter}}
+    spec:
+      serviceAccountName: connectivity-monitor
+      hostNetwork: true
+      priorityClassName: connectivity-exporter
```

Comment:
```suggestion
```

Since the priorityclass doesn't exist anymore the ds will have 0 desired replicas.